wpcom-deprecated-functions.php

```diff
@@ -18,3 +18,23 @@ function wpcom_vip_load_wp_rest_api() {
 function wpcom_vip_enable_https_canonical() {
 	_deprecated_function( __FUNCTION__, '2.0.0' );
 }
+
+/**
+ * Requires WordPress.com internal libraries
+ *
+ * This internal function is used in some WordPress.com themes. These shared libraries
+ * are no longer supported on VIP Go, so they will need to be copied directly into
+ * VIP Go client repositories as needed.
+ *
+ * @deprecated Not supported on VIP Go
+ */
+function require_lib( $slug ) {
+	_deprecated_function( __FUNCTION__, '2.0.0' );
+	
+	// Attempt to offer minimal back-compat.
+	// If the lib happens to exist in client-mu-plugins/lib, load it.
+	$lib = WPCOM_VIP_CLIENT_MU_PLUGIN_DIR . '/lib/' . $slug . '/' . $slug . '.php';
+	if ( file_exists( $lib ) ) {
```

Comment:
Should we check if `0 === validate_file( $lib )` as well?

Reply:
> Used to prevent directory traversal attacks, or to test a filename against a whitelist. 

I don't think that's necessary here?